app/process/watcher/crypto-transfer/crypto-transfer.go

```diff
@@ -1,3 +1,90 @@
 package crypto_transfer
 
-// crypto transfer watcher
+import (
+	"encoding/json"
+	"fmt"
+	hederasdk "github.com/hashgraph/hedera-sdk-go"
+	"github.com/limechain/hedera-eth-bridge-validator/app/process/model/essential"
+	"github.com/limechain/hedera-eth-bridge-validator/app/process/model/transaction"
+	"github.com/limechain/hedera-eth-bridge-validator/config"
+	"github.com/limechain/hedera-watcher-sdk/queue"
+	"github.com/limechain/hedera-watcher-sdk/types"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+type CryptoTransferWatcher struct {
```

Comment:
Add NewCryptoTransferWatcher, which takes httpClient, accountId and  typeMessage(to which the watcher will push messages ) as parameters and returns a *CryptoTransferWatcher